docs/button.rst

```diff
@@ -0,0 +1,77 @@
+
+.. _button:
+
+======
+Button
+======
+
+.. php:namespace:: atk4\ui\Button
+
+.. php:class:: Button
+
+Implements a clickable button::
+
+    $button = $view->add(new \atk4\ui\Button('Click me'));
+
+Button will typically inherit all same properties of a :php:class:`View`. Functionality
+of View alone yields in many various usage patterns such as::
+
+    $b1 = new Button(['Load', 'primary']);
+
+    $button = new Button('Hello there');
+    $button->addClass(['size big'=>true]);
```

Comment:
do we really need to write like this?
will `$button->addClass('size big')` not work?